source/backend/api/Repositories/Mayan/MayanDocumentRepository.cs

```diff
@@ -170,26 +172,14 @@ public async Task<ExternalResponse<QueryResponse<DocumentMetadataModel>>> TryGet
         public async Task<ExternalResponse<FileDownloadResponse>> TryDownloadFileAsync(long documentId, long fileId)
         {
             _logger.LogDebug("Downloading file {documentId}, {fileId}...", documentId, fileId);
-            string authenticationToken = await _authRepository.GetTokenAsync();
-
-            using HttpClient client = _httpClientFactory.CreateClient("Pims.Api.Logging");
-            client.DefaultRequestHeaders.Accept.Clear();
-            AddAuthentication(client, authenticationToken);
-            client.DefaultRequestHeaders.Accept.Add(new MediaTypeWithQualityHeaderValue(MediaTypeNames.Application.Json));
-
-            ExternalResponse<FileDownloadResponse> result = new ExternalResponse<FileDownloadResponse>()
+            ExternalResponse<FileDownloadResponse> result = new()
             {
                 Status = ExternalResponseStatus.Error,
             };
 
             try
             {
-                Uri endpoint = new($"{this._config.BaseUri}/documents/{documentId}/files/{fileId}/download/");
-                HttpResponseMessage httpResponse = await client.GetAsync(endpoint).ConfigureAwait(true);
-
-                var response = await ProcessDownloadResponse(httpResponse);
-
-                return response;
+                return await ProcessDownloadResponse(await GetFileAsync(documentId, fileId));
```

Comment:
nit: for debugging purposes, it would be better to have the result on a different variable and use that on the function.
```
var response = await GetFileAsync(documentId, fileId);
return await ProcessDownloadResponse(response);